resources/js/components/sections/bookshelfAddStep1.vue

```diff
@@ -33,8 +34,16 @@ export default {
   data () {
     return {
       searchText: ''
+      
+
     }
   },
+  beforeDestroy(){
+    this.reset()
```

Comment:
we should call `this.resetSearch()` instead than just reset, so that also the text input is cleaned

Reply:
Yeah . I'll do that. I thought by default it would be cleaned and just used this.reset(). I'll change it.